step/src/main/webapp/infoForm.jsp

```diff
@@ -31,7 +31,6 @@ limitations under the License.
   </head>
   <body>
     <nav>
-        <a href="<%= logoutURL %>"> Log Out </a>
```

Comment:
Hm, yeah, I guess it's probably good for the user to be able to back out mid-way. Will change it to say 'home' as opposed to 'log out' though since that feels like better UX